dev/tidb-in-kubernetes/maintain/lightning.md

```diff
@@ -0,0 +1,119 @@
+
+        i. Ensure `dataSource.local.nodeName` and `dataSource.local.hostPath` are commented out.
+
+        ii. Create a `Secret` containing the rclone configuration. A sample configuration is listed below, only one cloud storage configuration is required. For other cloud storage, please refer to rclone [documentation](https://rclone.org/).
+
+            {{< copyable "" >}}
+
+            ```yaml
+            apiVersion: v1
+            kind: Secret
+            metadata:
+              name: cloud-storage-secret
+            type: Opaque
+            stringData:
+              rclone.conf: |
+                [s3]
+                type = s3
+                provider = AWS
+                env_auth = false
+                access_key_id = <my-access-key>
+                secret_access_key = <my-secret-key>
+                region = us-east-1
+
+                [ceph]
+                type = s3
+                provider = Ceph
+                env_auth = false
+                access_key_id = <my-access-key>
+                secret_access_key = <my-secret-key>
+                endpoint = <ceph-object-store-endpoint>
+                region = :default-placement
+
+                [gcs]
+                type = google cloud storage
+                # The service account must include Storage Object Viewer role
+                # The content can be retrieved by `cat <service-account-file.json> | jq -c .`
+                service_account_credentials = <service-account-json-file-content>
+            ```
+
+            Fill in the placeholders with your configurations and save it as `secret.yaml`, and then create the secret via `kubectl apply -f secret.yaml -n <namespace>`
+
+        iii. Configure the `dataSource.remote.storageClassName` to an existing storage class in the Kubernetes cluster.
+
+2. Deploy TiDB Lightning
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    helm install pingcap/tidb-lightning --name=<release-name> --namespace=<namespace> -f values.yaml --version=<chart-version>
+    ```
+
+When TiDB Lightning fails, it cannot simply be restarted, manual intervention is required. TiDB Lightning chart has hack script which avoids pod exiting and restarting when it fails. This makes it handy to do manual intervention, however users have to check the pod's log instead of simply checking the pod's status to determine if the restore job failed or not. This behavior can be disabled by setting `failFast` to `true` in `values.yaml`.
```

Comment:
```suggestion
When TiDB Lightning fails, it cannot simply be restarted, but manual intervention is required. TiDB Lightning chart has a hack script which avoids the pod exiting and restarting when it fails. This makes it handy to do manual intervention, no matter how users have to check the pod's log instead of simply checking the pod's status to determine if the `restore` job failed or not. This behavior can be disabled by setting `failFast` to `true` in `values.yaml`.
```